lib/linter-reek.js

```diff
@@ -90,13 +90,15 @@ export default {
         while (match !== null) {
           const line = Number.parseInt(match[1], 10) - 1;
           const urlBase = 'https://github.com/troessner/reek/blob/master/docs/';
-          const ruleLink = `[<a href="${urlBase}${match[3]}.md">${match[3]}</a>]`;
+          const ruleLink = `${urlBase}${match[3]}.md`;
```

Comment:
Since this isn't the large mess that it was before you might as well just embed it below.